src/app/info/WikiItem.tsx

```diff
@@ -35,22 +38,24 @@ export const formatContent: (topRowContent: string) => (React.JSX.Element)[] = (
     return contentParts.map((part: ContentPart)=>{return part.href ? <a href={part.href} key={part.key}>{part.content}</a> : <span key={part.key}>{part.content}</span>})
 };
 
-const WikiItems: React.FC<AccordianProps> = (props) => {
-    const formattedContent = formatContent(props.row.content);
-    return (
-        <>
-            <Accordion>
-                <AccordionSummary
-                    expandIcon={<ExpandMoreIcon />}
-                    aria-controls="panel1-content"
-                    id="panel1-header"
-                >
-                    {props.row.title}
-                </AccordionSummary>
-                <AccordionDetails>{formattedContent}</AccordionDetails>
-            </Accordion>
-        </>
-    );
+const WikiItem: React.FC<AccordianProps> = (props) => {
+    const visibleWikiItem = useMemo(() => {
+        const formattedContent = formatContent(props.row.content);
```

Comment:
I've realised I was wrong here, we don't want use memo because this can be server side rendered. However, lets have a chat about use memo and its usage because this isn't quite right here.